azure-pipelines.yml

```diff
@@ -0,0 +1,156 @@
+# azure-pipelines build file
+# https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema
+
+# TODO: Add gh-pages deployment on tag trigger
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
+    - RB-*
+    - ci_test
+
+pr:
+  autoCancel: true
+  branches:
+    include:
+    - master
+    - RB-*
+
+# Common variables
+variables:
+  cxxStd: 14
+  nixCxxFlags: -std=c++$(cxxStd) -fPIC
+
+jobs:
+
+# ------------------------------------------------------------------------------
+# Linux
+# ------------------------------------------------------------------------------
+# TODO: Move Docker images to ASWF DockerHub account
+
+- job: Linux
+  strategy:
+    matrix:
+      Ubuntu 18.04 VFX CY2020:
+        agentImage: 'ubuntu-16.04'
+        containerImage: michdolan/ubuntu18.04:vfx2020_py27
+        cxxFlags: ''
+        cxxCompiler: g++
+        cCompiler: gcc
+      Ubuntu 18.04 VFX CY2019:
+        agentImage: 'ubuntu-16.04'
+        containerImage: michdolan/ubuntu18.04:vfx2019
+        cxxFlags: ''
+        cxxCompiler: g++
+        cCompiler: gcc
+      Ubuntu 18.04 Clang 6.0:
+        agentImage: 'ubuntu-16.04'
+        containerImage: michdolan/ubuntu18.04:vfx2019
+        cxxFlags: -Wno-deprecated-register
```

Comment:
It would be better on long-term to add the flag `-Wno-deprecated-register` in the CMakeLists.txt

Reply:
Sounds good.